testhelper/docker/resource/registry/registry.go

```diff
@@ -0,0 +1,43 @@
+package registry
+
+import (
+	"fmt"
+	"os"
+
+	dc "github.com/ory/dockertest/v3/docker"
+)
+
+type RegistryConfig struct {
+	URL      string
+	Username string
+	Password string
+}
+
+// NewRegistry creates a registry configuration that uses a mirror if configured, otherwise Docker Hub
+func NewRegistry() *RegistryConfig {
```

Comment:
we don't really need a NewRegistry, simply 2 package functions:
- registry.ImagePath
- registry.AuthConfiguration